examples/std-transaction-signing/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"encoding/hex"
+	"log"
+
+	"github.com/flashbots/suapp-examples/framework"
+)
+
+func main() {
+	fr := framework.New()
+	priv := "b71c71a67e1177ad4e901695e1b4b9ee17ae16c6668d313eac2f96dbcda3f291"
+
+	contract := fr.Suave.DeployContract("transaction-signing.sol/TransactionSigning.json")
+	receipt := contract.SendTransaction("example", nil, []byte(priv))
+
+	// validate the signature
+	txnSignatureEvent, _ := contract.Abi.Events["TxnSignature"].ParseLog(receipt.Logs[0])
```

Comment:
I get nervous when I see ignored errors. Can we at least catch this and panic?  I want to guard against the case where the ABI changes in the future.